src/components/Match/Chat/Chat.jsx

```diff
@@ -457,7 +457,7 @@ class Chat extends React.Component {
             return (
               <li
                 id={index}
-                key={index}
+                key={msg.key}
```

Comment:
key here is the message text and is not unique.

Actually it's kind of difficult to come up with a unique key here that doesn't use index at all, since the same player may say the same thing in the same second.

Reply:
I wonder if in cases where finding a unique key is difficult we can use the uuid library to generate keys

Reply:
I don't think that will work well since it'll generate a new key every time the component re-renders